legal-api/src/legal_api/models/party.py

```diff
@@ -32,7 +32,7 @@ class PartyTypes(Enum):
         """Render an Enum of the party types."""
 
         PERSON = 'person'
-        ORGANIZATION = 'organization'
+        ORGANIZATION = 'org'
```

Comment:
maybe we should change the schema to match ?
checking schema and they spell out the full item **organization**

Reply:
If we change the schema, then we need to change the UIs too. If we do really need to change 'org' to 'organization' for some reason, I'm ok changing the schema now. Otherwise I would leave it as it is now.

Reply:
This conversation is outdated.